scripts/gen_profiles_tests.py

```diff
@@ -156,8 +156,8 @@ class TestsCapabilitiesGenerated : public VkTestFramework {
 class ProfileGenerator():
     i = 1
     skipped_features = []
-    skipped_members = ["sType", "pNext", "physicalDevices", "driverID"]
-    skipped_properties_structs = ["VkPhysicalDeviceHostImageCopyPropertiesEXT", "VkPhysicalDeviceLineRasterizationPropertiesEXT", "VkPhysicalDeviceLayeredApiPropertiesListKHR"]
+    skipped_members = ["sType", "pNext", "physicalDevices", "driverID", "copySrcLayoutCount", "copyDstLayoutCount"]
```

Comment:
The one that needs skipping are `pCopySrcLayouts` and `pCopyDstLayouts` I believe.

Or am I missing something?

Reply:
No, `copySrcLayoutCount` and `copyDstLayoutCount` are ignored, you will only set `pCopySrcLayouts` and `pCopyDstLayouts` when using host image copy in a profile. 

For example it will be:
```
				"VkPhysicalDeviceHostImageCopyPropertiesEXT": {
					"pCopySrcLayouts": [
						"VK_IMAGE_LAYOUT_GENERAL",
						"VK_IMAGE_LAYOUT_SHADER_READ_ONLY_OPTIMAL",
						"VK_IMAGE_LAYOUT_TRANSFER_SRC_OPTIMAL"
					],
					"pCopyDstLayouts": [
						"VK_IMAGE_LAYOUT_GENERAL",
						"VK_IMAGE_LAYOUT_SHADER_READ_ONLY_OPTIMAL",
						"VK_IMAGE_LAYOUT_TRANSFER_DST_OPTIMAL"
					],
					...
```